speed-dreams/source-2.2.3/data/data/menu/music/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+INCLUDE(../../../cmake/macros.cmake)
+
+
+
+FILE(GLOB OGG_FILES *.ogg)
+
+
+
+SD_INSTALL_FILES(DATA data/music FILES ${OGG_FILES})
+
+
```

Comment:
Not sure why this is in here?

Reply:
I am also not sure why it is in there I didn't add it on purpose/consiously